app/src/main/java/com/readrops/app/more/preferences/PreferencesScreen.kt

```diff
@@ -210,6 +218,36 @@ class PreferencesScreen : AndroidScreen() {
                                 title = stringResource(id = R.string.open_items_in),
                                 onValueChange = {}
                             )
+
+                            var showDialog by remember { mutableStateOf(false) }
+                            val defaultItem = Item(
+                                title = stringResource(R.string.example_item_title),
+                                author = stringResource(R.string.example_item_author),
+                                content = stringResource(R.string.example_item_content),
+                                link = "https://example.org"
+
+                            )
+                            val exampleItem by produceState(defaultItem) {
```

Comment:
Same thing, please put this in `PreferencesScreenModel` and avoid making database requests directly in the UI.